packages/cli/src/generators/index.js

```diff
@@ -0,0 +1,156 @@
+const path = require('path');
+
+const filter = require('gulp-filter');
+const prettier = require('gulp-prettier');
+
+const {
+  NODE_VERSION,
+  PACKAGE_VERSION,
+  PLATFORM_PACKAGE
+} = require('../constants');
+
+const Generator = require('yeoman-generator');
+
+const writeGenericReadme = gen => {
+  gen.fs.copyTpl(
+    gen.templatePath('README.template.md'),
+    gen.destinationPath('README.md'),
+    { name: gen.options.packageName }
+  );
+};
+
+const appendReadme = gen => {
+  const content = gen.fs.read(
+    gen.templatePath(gen.options.template, 'README.md'),
+    { defaults: '' }
+  );
+  if (content) {
+    gen.fs.append(gen.destinationPath('README.md'), '\n' + content);
+  }
+};
+
+const writeGitignore = gen => {
+  gen.fs.copy(gen.templatePath('gitignore'), gen.destinationPath('.gitignore'));
+};
+
+const writeGenericPackageJson = gen => {
+  gen.fs.writeJSON('package.json', {
+    name: gen.options.packageName,
+    version: '1.0.0',
+    description: '',
+    main: 'index.js',
+    scripts: {
+      test: 'jest'
+    },
+    engines: {
+      node: `>=${NODE_VERSION}`,
```

Comment:
honestly probably don't need this either - it mostly causes confusion. We enforce the node version w/ the CLI and their core version sets it for runtime. Plus it's [not even enforced](https://docs.npmjs.com/files/package.json#engines).